python/pyarrow/tests/test_schema.py

```diff
@@ -725,6 +725,10 @@ def test_schema_merge():
     result = pa.unify_schemas((a, b, c))
     assert result.equals(expected)
 
+    result = pa.unify_schemas(
+        [b, d], options=pa.FieldMergeOptions.permissive())
+    assert result.equals(d)
```

Comment:
It would be nice if I could also write:
```suggestion
    result = pa.unify_schemas(
        [b, d], options=pa.FieldMergeOptions.permissive())
    assert result.equals(d)
    result = pa.unify_schemas([b, d], options="permissive")
    assert result.equals(d)
```

@jorisvandenbossche What do you think?

Reply:
I like it, added it!